feat: add captcha native app criteria

marshcooke opened the pull request:
Link native app criteria review checklist:

- [ ] Run the project
- [ ]  Cross compare the old & new side by side, note anything missing
- [ ]  Note if anything is styled oddly or missing or broken
- [ ]  Read the new criteria file, note anywhere in the content where edits or changes are needed

danbivins commented on public/content/documentation/native/controls/captcha.md:
```diff
@@ -0,0 +1,288 @@
+    1. A non-interactive image in a table row, where the whole table row acts like a button.  
+    2. A single interactive element, like an image button
+    3. An audio challenge as an equivalent experience to the image challenge
+- When selecting the type of Captcha, do not limit the options to only <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html#dfn-cognitive-function-test"> cognitive function tests</a> (e.g. remembering or transcribing a word, or recognizing a picture the website provided).  When possible, leverage <a href="https://www.w3.org/WAI/WCAG22/Understanding/accessibl ...
```
an aria-label is being added when I view in the browser but I don't see it here ... the aria-label seems a bit lengthy with the full url spelled out. Is this something being done at a template-level or am I missing something here?

Also, I'm trying to figure out why these links don't have an underline though it doesn't seem to be anything you're doing.

danbivins replied:
Ah, for the second issue I noted there's a style `.MagentaA11y__content-details a` that is overriding the normal link underline.

marshcooke replied:
@silverli - do you have any insight to the aria-label situation happening? 

@danbivins - is there any update on the underlining not populating that I missed Thurs/Fri?

danbivins replied:
This is a global issue that Lindsay will fix.

danbivins commented on public/content/documentation/native/controls/captcha.md:
```diff
@@ -0,0 +1,288 @@
+    2. Name describes the purpose of the control and matches any visible text in image
+    3. Programmatic name of each interactive component in audio challenge is announced
+
+- **Android Views**  
```
I think these should be h4

marshcooke commented:
Updated gherkin section (section/scenarios, structure and layout), updated some link spacing, updated the h4s that didn't port in correctly from raw HTML the first time, also removed some "  " (double spaces) between words, colors and words, and ends of sentences and words. 

Re-requested a review from @danbivins; also outstanding question about links in comments to Lindsay and Dan.

danbivins reviewed (approved):
looks good to me!
